method/index.js

```diff
@@ -1,20 +1,52 @@
+const Crut = require('ssb-crut')
 const Init = require('./group/init')
 const AddMember = require('./group/add-member')
-const { CreateFeedGroupLink, CreateGroupSubgroupLink } = require('./link/create')
 const FindByGroupByFeedId = require('./link/find-group-by-feedid')
 const Application = require('./application')
+const FeedGroupLink = require('../spec/link/feed-group')
+const GroupSubgroupLink = require('../spec/link/group-subgroup')
 
 module.exports = function Method (ssb, keystore, state) {
   const application = Application(ssb)
+  const linkFeedCrut = new Crut(ssb, FeedGroupLink)
+  const linkSubgroupCrut = new Crut(ssb, GroupSubgroupLink)
 
   return {
     group: {
       init: patient(Init(ssb, keystore, state)),
       addMember: patient(AddMember(ssb, keystore, state))
     },
     link: {
-      create: patient(CreateFeedGroupLink(ssb)),
-      createSubgroupLink: patient(CreateGroupSubgroupLink(ssb)),
+      create ({ group, name }, cb) {
+        const input = {
+          parent: ssb.id,
+          child: group,
+          name,
+          recps: [group]
+        }
+
+        // set the recps on the link to be the same as the recps on the settings
+        if (!name) delete input.name
+
+        patient(linkFeedCrut.create.bind(linkFeedCrut)(input, (err, linkId) => {
+          if (err) return cb(err)
+
+          linkFeedCrut.read(linkId, cb)
+        }))
+      },
+      createSubgroupLink: ({ group, subgroup }, cb) => {
+        const input = {
+          parent: group,
+          child: subgroup,
+          recps: [group]
```

Comment:
Didnt bother to add any checks to see if the recps only had one recp because its hardcoded in there